runtime/near-vm-logic/src/config.rs

```diff
@@ -486,6 +487,124 @@ pub enum ExtCosts {
     validator_total_stake_base,
 }
 
+// Type of an action, used in fees logic.
+#[derive(Copy, Clone, Hash, PartialEq, Eq, Debug, PartialOrd, Ord)]
+#[allow(non_camel_case_types)]
+pub enum Actions {
```

Comment:
I don't like the name of it, because it's confusing to `Action`. Is it possible to rename it to `ActionCosts` or `ActionProfiler` or something similar to indicate it's not an `Action`

Reply:
Also it doesn't have to be public. Can be `pub(crate)`

Reply:
It's exported to standalone runner, so not sure if could be crate pub.

Reply:
This type is used outside of the crate, namely in standalone launcher.